tests/api-testing/tests/test_cipher.py

```
@@ -0,0 +1,584 @@
+    # Attempt to create the cipher key
+        resp_create_cipher_simpleOO = session.post(
+            f"{base_url}api/{org_id}/cipher_keys", json=payload_simpleOO
+        )
+        print(resp_create_cipher_simpleOO.content)
+
+        if resp_create_cipher_simpleOO.status_code == 200:
+            break  # Exit the loop if the key is created successfully
+        elif resp_create_cipher_simpleOO.status_code == 400:
+            continue  # Key already exists, try again
+        else:
+            assert False, f"Unexpected error: {resp_create_cipher_simpleOO.status_code} {resp_create_cipher_simpleOO.content}"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)